src/app/Incons/NavigationButtonBackIcon.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+export default function NavigationButtonBackIncon(): JSX.Element {
```

Comment:
There is one "n" too much ;)
Anyway: If you are going to use this Icon somewhere else, the name should be more general, like "BackIcon".
The directory also is called "Incons" with an "n".
```suggestion
export default function NavigationButtonBackIcon(): JSX.Element {
```